HippUnfoldHCP/PostHippUnfoldHCP.sh

```diff
@@ -0,0 +1,300 @@
+#!/bin/bash
+set -eu
+pipedirguessed=0
+if [[ "${HCPPIPEDIR:-}" == "" ]]
+then
+  # pipedirguessed=1
+   #fix this if the script is more than one level below HCPPIPEDIR
+   export HCPPIPEDIR="$(dirname -- "$0")/.."
+fi
+
+source "$HCPPIPEDIR/global/scripts/newopts.shlib" "$@"
+source "$HCPPIPEDIR/global/scripts/debug.shlib" "$@"
+
+
+opts_SetScriptDescription "Make some BIDS structures and run HippUnfold"
+
+opts_AddMandatory '--study-folder' 'StudyFolder' 'path' "folder containing all subjects"
+opts_AddMandatory '--subject' 'Subject' 'subject ID' ""
```

Comment:
This was changed to Session during the refactoring?

Reply:
I think some of those changes might have been overzealous.  In any case, this is a cross-sectional pipeline at the moment and I have not thought at all about making it longitudinal.  That might be something that HippUnfold would need to internally do.  We rely on FreeSurfer to setup the longitudinal processing for cerebral cortex.